openstudiocore/src/utilities/filetypes/EpwFile.cpp

```diff
@@ -132,67 +132,119 @@ EpwDataPoint::EpwDataPoint(int year,int month,int day,int hour,int minute,
   setLiquidPrecipitationQuantity(liquidPrecipitationQuantity);
 }
 
-boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(std::string line)
+boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(const std::string &line)
+{
+  std::vector<std::string> list;
+  boost::split(list, line, boost::is_any_of(","));

```

Comment:
@jasondegraw my vote wasn't for regex, especially since I don't think we can use it... https://stackoverflow.com/questions/8561850/compile-stdregex-iterator-with-gcc/8913707#8913707

I'd rather use ...
http://stackoverflow.com/a/9436872/2358662
or 
http://stackoverflow.com/a/18564118/2358662